Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -78,10 +78,14 @@
 static NSString * const backgroundContentEventListenersVersionKey = @"BackgroundContentEventListenersVersion";
 static NSString * const lastSeenBaseURLStateKey = @"LastSeenBaseURL";
 static NSString * const lastSeenVersionStateKey = @"LastSeenVersion";
+static NSString * const lastLoadedDNRHashStateKey = @"LastLoadedDNRHash";
 
 // Update this value when any changes are made to the WebExtensionEventListenerType enum.
 static constexpr NSInteger currentBackgroundContentListenerStateVersion = 2;
 
+// Update this value when any changes are made to the rule translation logic in _WKWebExtensionDeclarativeNetRequestRule.
+static NSString * const currentDeclarativeNetRequestRuleTranslatorVersion = @"1";
```

Comment:
I think a `size_t` or other integer type for this would be cleaner, since we are already using a format string to append it.

Reply:
Or `NSInteger` to match the one above.